### DISCUSSION TITLE
Can we access firefox theme colors when styling tridactyl themes?

### DISCUSSION THREAD

**Discussion by davidscotson** [tridactyl/tridactyl#4110] [Q&A]
I'm assuming the answer to this is no, for security reasons, to stop websites and/or web extensions pretending to be core parts of your browser, but wanted to check as I can't find that stated clearly anywhere.

I intend to use the colors from the default dark theme listed here:
https://searchfox.org/mozilla-release/source/browser/themes/addons/dark/manifest.json#18

by simply copying them into a tridactyl theme, but if there's a dynamic way to reference them so it picks up whatever color the user has set with their theme, then I'd rather use that.

**Comment by bovine3dom** [comment]
Hey, sorry this took so long to reply to, I only just noticed it :)

I don't think this is possible. We could give Tridactyl the `theme` permission and then you could use `browser.theme.getCurrent()`, but as far as I can tell that only works for themes that have been added using an extension.
